src/types/backend.ts

```diff
@@ -0,0 +1,12 @@
+export interface SubmitPaymentParams {
+  sourceAccount: string
+  destinationAccount: string
+  sourceAmount: string
+  destinationAmount: string
+}
+
+export interface IBackend {
```

Comment:
What does `IBackend` mean?

Reply:
Interfaces are commonly prefixed with `I` in many object-oriented languages, particularly .NET. I guess .NET developers brought the practice to TypeScript.

You can [go here](https://github.com/Microsoft/TypeScript-Handbook/issues/121) to read a bunch of people debating the pros and cons of prefixing interfaces with `I`.

I'll get rid of the `I`, there isn't any strong reason we need it and if it's confusing for you, it's probably going to be confusing for others.